raiden_contracts/contract_manager.py

```diff
@@ -79,7 +79,8 @@ def __init__(self, path: Path) -> None:
 
     def get_contract(self, contract_name: str) -> CompiledContract:
         """ Return ABI, BIN of the given contract. """
-        assert self.contracts, "ContractManager should have contracts compiled"
+        if not self.contracts:
```

Comment:
Assertion was right here.  The constructor fails unless `self.contracts` has something.